scripts/scripts.js

```diff
@@ -530,10 +533,57 @@ async function loadEager(doc) {
   }
 }
 
+function setUpSoftNavigation() {
+  document.body.addEventListener('click', async (e) => {
+    const link = e.target.closest('a');
+    if (link && getMetadata('template') === 'guides' && e.target.closest('.side-navigation')) {
+      const { href } = link;
+      const hrefURL = new URL(href);
+      if (hrefURL.origin === window.location.origin) {
+        e.preventDefault();
+        try {
+          const resp = await fetch(href);
+          const html = await resp.text();
+          const dom = new DOMParser().parseFromString(html, 'text/html');
+          const main = dom.querySelector('main');
+          const template = dom.querySelector('meta[name="template"]');
+          if (template && template.getAttribute('content') === 'guides') {
+            await decorateMain(main);
+            await loadBlocks(main);
+            const currentMain = document.querySelector('main');
+            const children = [...currentMain.children].slice(2);
+            sampleRUM('leave');
+            children.forEach((child) => child.remove());
+            while (main.firstElementChild) currentMain.append(main.firstElementChild);
+            const title = dom.querySelector('title').textContent;
+            const category = dom.querySelector('meta[name="category"').getAttribute('content');
+            document.querySelector('meta[name="category"]').setAttribute('content', category);
+            document.querySelector('meta[property="og:title"]').setAttribute('content', title);
+            document.querySelector('title').textContent = title;
+            decorateBreadcrumb(currentMain);
+            const oldhref = window.location.href.pathname;
```

Comment:
@trieloff shouldn't this be `window.location.pathname` ?

Reply:
```suggestion
            const oldhref = window.location.pathname;
```

yes, it should.

Reply:
The variable name is confusing, too 😉

Reply:
Seeing ` window.location.href.pathname` gives me the feeling this has never been tested...